src/Routes/PrivateRoute.tsx

```diff
@@ -1,16 +1,13 @@
-import { getConfig } from 'config'
 import { Redirect, Route, RouteProps } from 'react-router-dom'
 
 type PrivateRouteProps = {
   hasWallet: boolean
 } & RouteProps
 
-const HIDE_SPLASH = getConfig().REACT_APP_HIDE_SPLASH
-
 export const PrivateRoute = ({ hasWallet, ...rest }: PrivateRouteProps) => {
   const { location } = rest
 
-  return hasWallet || HIDE_SPLASH ? (
```

Comment:
`REACT_APP_HIDE_SPLASH` is referenced, you just removed the reference. What's the reason for removing the reference?

Reply:
with the auto wallet reconnect, and metamask, the issue this feature (splash screen) solved is no longer useful, we don't use it, so we're removing it